src/components/SMARTBox/PatientBox.js

```diff
@@ -400,9 +397,9 @@ export default class PatientBox extends Component {
 
   /**
    * Launch In progress From
-  */
+   */
```

Comment:
should say "Form" above (not touched by this PR, so GitHub won't let me suggest it)